### PR TITLE
Fix NOTES indentation

### PR DESCRIPTION
**Pull request by lindhe**
Previously, the installation notes (NOTES.txt) had inconsistent indentation making it hard to read sometimes. This commit fixes that.

I also chose to remove the proceeding whitespaces so that the commands are easier to select with the mouse in order to copy-paste them.

**Before:**
Before
**After**
After

### CODE REVIEW

**Comment by hbagdi** [comment]
Can you rebase on `next` branch?

**Comment by lindhe** [comment]
How come there are non-related commits in this PR now?

**Comment by hbagdi** [comment]
Please rebase your commit on top of next branch and then force push.

**Comment by lindhe** [comment]
Should I bump the Chart version number, or is that up to someone else?

**Comment by hbagdi** [comment]
Don't bump up the chart version. We(maintainers) will take care of it later.

**Comment by lindhe** [comment]
Got ya. Feel free to change that, or wait until tomorrow and I'll fix it.

File: charts/kong/Chart.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ maintainers:
   email: harry@konghq.com
 name: kong
 sources:
-version: 1.2.0
+version: 1.2.1
```
Comment (hbagdi) [inline charts/kong/Chart.yaml:13]: Please remove the version bump. We(maintainers) will take care of it later.

**Comment by rainest** [comment]
I added similar changes when touching NOTES.txt for the new content added by https://github.com/Kong/charts/pull/55/files#diff-d8fc77ab0c592f56e664e30251df0da7

**Comment by hbagdi** [comment]
@rainest, please incorporate all the changes from this PR into yours and we can close this once your PR is merged in.

**Comment by hbagdi** [comment]
This is being taken care of in #55

**Pull request closed without merging**